python/lance/pytorch/data.py

```diff
@@ -43,6 +43,8 @@ def _data_to_tensor(data: Any) -> Union[torch.Tensor, PIL.Image.Image]:
         return data.to_pil()
     elif isinstance(data, dict):
         return {k: to_tensor(v) for k, v in data.items()}
+    elif isinstance(data, str):
```

Comment:
What is the use case for this? Pytorch model can not take in strings, no valid `torch.Tensor` type for string.

Reply:
so say you have `LanceDataset(uri, filter=pc.field("split") == "train")`

then the split column is included in the dataset and in record mode we get "train" as the value.